provisiondomainmanager_policy.tf

```diff
@@ -240,6 +237,11 @@ data "aws_iam_policy_document" "provisiondomainmanager_policy_ecr_ecs_elb_events
       "ecs:DescribeTaskDefinition",
```

Comment:
Can we use tags to restrict these actions?

Reply:
@zenine07 - Two comments I made in the original PR showed up in the wrong place.  I recreated one of them here (in the right place I hope) but I can't find where the other one was.

But my essential question is in this PR's main comment - have you and @dav3r already identified every place where permissions can be restricted - either by resource IDs or tags?

Reply:
@jsf9k Yes, we have restricted them as much as we can unfortunately. I wish we could restrict more, but this app is not a very conventional app and AWS doesn't have support for tag-based access for every service. And the ones that don't have the tag-based support are ones that we cannot predict what the resource IDs will be, such as S3 bucket names or hosted zone ids.